install/kubernetes/cilium/values.yaml

```diff
@@ -2812,8 +2813,6 @@ preflight:
     - key: node.cloudprovider.kubernetes.io/uninitialized
       effect: NoSchedule
       value: "true"
```

Comment:
I also wonder if we want to remove those tolerations as well to match cilium-agent more closely

Reply:
emm.. not sure it will break the current behavior during the preflight process... but I am happy to commit the changes if needed.